applications/TrilinosApplication/custom_utilities/trilinos_cpp_test_utilities.cpp

```diff
@@ -270,13 +276,18 @@ void TrilinosCPPTestUtilities::CheckSparseMatrix(
                     const int col_gid = rA.ColMap().GID(cols[j]);
                     if (col_gid == column) {
                         KRATOS_CHECK_RELATIVE_NEAR(value, vals[j], Tolerance)
+                        ++local_validated_values;
                         break;
                     }
                 }
                 break;
             }
         }
     }
+
+    // Checking that all the values has been validated
+    const std::size_t global_validated_values = rDataCommunicator.SumAll(local_validated_values);
```

Comment:
you can do that also with the epetra comm (basically the same concept as our data comm)

Reply:
Ours is easier to use :)

Reply:
this way you could avoid passing of the additional argument
same as you did with other functions

Reply:
it also has the [SumAll](https://docs.trilinos.org/dev/packages/epetra/doc/html/classEpetra__Comm.html#af81a51a85a8756371c6005de62378dbb) function

No strong opinion as this is only for testing

Reply:
i agree with @philbucher on this